test/integration/targets/ansible-playbook-callbacks/all-callbacks.yml

```diff
@@ -96,26 +96,22 @@
       ignore_errors: true
 
     - name: async poll ok
-      command: sleep 2
```

Comment:
I don't think this is needed as I have yet to see this being an issue in our CI. Generally we want the sleep to be as short as possible to reduce tests running time.

Reply:
I definitely hit the issue locally, and I saw it in CI once too, but that may have been with an earlier iteration of the values. Either way, it should be possible to run the integration tests on a laptop reliably.

I understand the concern around test runtime, and set it up to ensure that only one poll callback is sent per task.

The problem is that the sleep duration is a multiple of the poll interval, so the number of polls can vary. I chose these numbers to make the sleep finish in the middle of the 1st and 2nd poll.

I'll try halving all of the numbers, which should reduce the task duration to 2s, as it was before.

Reply:
I'm testing this locally like this:

```
for i in $(seq 1 10); do if ! ansible-test integration ansible-playbook-callbacks; then echo "Failed on $i"; break; fi; done
```

With the original numbers (sleep 2 poll 1) and it failed on the 2nd attempt. Tried again and it failed on the 5th attempt.

I tried it twice with the change proposed here (sleep 3 poll 2) and it did not fail.

I tried it with the numbers halved (sleep 1.5 poll 1) and it failed on the second attempt.

I tried a few other combinations, but couldn't find something that worked well (poll must be an integer).

So I'm inclined to keep the numbers proposed here - a slow test is better than an unreliable one.